mapping_test_data = [
    {
        "product": "python",
        "version": "2.7.0",
        "version_strings": [
            "2.7.0",
            "Internal error in the Python interpreter",
            "Cpython",
            "lib/python2.7",
            "~/anaconda/bin/python",
        ],
    },
    {
        "product": "python",
        "version": "3.7.1",
        "version_strings": [
            "3.7.1",
            "Internal error in the Python interpreter",
            "lib/python3.7",
        ],
    },
]
package_test_data = [
    {
        "url": "https://kojipkgs.fedoraproject.org//packages/python3/3.8.2~rc1/1.fc33/aarch64/",
        "package_name": "python3-3.8.2~rc1-1.fc33.aarch64.rpm",
        "product": "python",
        "version": "3.8.2",
    },
    {
        "url": "https://rpmfind.net/linux/openmandriva/4.0/repository/x86_64/main/release/",
        "package_name": "python-3.7.3-4-omv4000.x86_64.rpm",
        "product": "python",
        "version": "3.7.3",
    },
]
